ceph-windows-test/build/utils.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o pipefail
+
+CEPH_WINDOWS_ZIP=${CEPH_WINDOWS_ZIP:-"$WORKSPACE/ceph.zip"}
+
+VM_NAME="ceph-win-ltsc2019-${BUILD_ID}"
```

Comment:
You may not want to put `${BUILD_ID}` at the end;  Or maybe add a character or something after it.  I've run into issues in the past where the Build ID was 1, for example, and then all builds starting with 1 were also affected.

Reply:
I can change the name to `ceph-win-ltsc2019-${JOB_NAME}-${BUILD_ID}`.

Adding `${JOB_NAME}` as part of the name should assure that we get an unique name for the VM.

Reply:
The `${BUILD_ID}` being at the end of the VM name is the problem.

`$ grep ceph-win-ltsc2019-1` will return results for:
```
ceph-win-ltsc2019-1
ceph-win-ltsc2019-12
ceph-win-ltsc2019-123
etc.
```

Reply:
Nevermind.  You're not grepping anywhere.  Apologies.

Reply:
> Nevermind. You're not grepping anywhere. Apologies.

I grep in the `cleanup_windows_tests_env` function to delete the virsh VM.